src/autocomplete.js

```diff
@@ -21,26 +23,34 @@ var config = require("./config");
  * @property {string} [docText] - a plain text that would be displayed as an additional popup. If `docHTML` exists,
  * it would be used instead of `docText`.
  * @property {string} [completerId] - the identifier of the completer
- * @property {Ace.Range} [range] - An object specifying the range of text to be replaced with the new completion value (experimental)
+ * @property {import("../ace").Ace.IRange} [range] - An object specifying the range of text to be replaced with the new completion value (experimental)
```

Comment:
why do we need this change and `IRange`

Reply:
At times, we expect only objects with the {start, end} parameters, not instances of `Range`. This is one of those instances. Additionally, I've replaced `Range` with `IRange` everywhere we do not expect an instance of `Range`. For example, see  https://github.com/mkslanc/ace/blob/better-types/src/document.js#L174C5-L174C21